source/_integrations/slack.markdown

```diff
@@ -162,6 +162,16 @@ To include a remote file with the Slack message, use these attributes underneath
 | `username`               |      yes | An optional username if the URL is protected by HTTP Basic Auth.
 | `password`               |      yes | An optional password if the URL is protected by HTTP Basic Auth.
 
+### Obtaining a User ID
+
+Some of the examples below use a member ID. This is a unique string assigned by Slack and not a username set by the user. To get a user ID:
+
+1. Select a Slack user (both name and profile image work) to bring up their profile side panel
+2. Open the context menu by selecting the three dots
+3. Select `Copy member ID`
```

Comment:
```suggestion
1. Select a Slack user (both name and profile image work) to bring up their profile side panel.
2. Open the context menu by selecting the three dots.
3. Select **Copy member ID**.
```
- Added periods because these are complete sentences. See [MSG](https://learn.microsoft.com/en-us/style-guide/punctuation/periods)
- We use bold to markup UI elements
What is the difference between user and member ID? Can we stick with either one? Or would this be incorrect?